### ISSUE TITLE
[Block] Social - ability to add custom icons are needed

### ISSUE THREAD

**Issue by palomnik** [WordPress/gutenberg#39868]
There are some other requests to add icons for discord and the like.  I would also like to add the icon for Facebook Groups.  It would be good if users could add their own custom icons to the block.  You could specify size and svg for uploads.

**Comment by dozentis** [comment]
**Social Icon block**
I think a phone icon (for links like tel:+.....) is neccessary

**Comment by eric-michel** [comment]
I came to create an issue about the same thing. There are a lot of icons that could be added - allowing for custom icons to be uploaded would be a great start for more flexibility.

**Comment by YogieAnamCara** [comment]
Great idea, I like to upvote. I need a few other social icons and the fastest and best way would be to be able to add own icons to links.
Currently I am missing VERO

**Comment by Azragh** [comment]
Yup. Just was looking for a way to do this but doesn't seem possible. There are so many country or language specific social networks and platforms, its kinda sad that this block only can be used for the big ones. In more than half of the websites I develop, the client needs a Xing icon - but since the platform is only used in German-speaking countries, I still have to resort to custom HTML (and my custom SVG icon shortcodes), which is anything but user-friendly by comparison.

**Comment by scrobbleme** [comment]
It is quite annoying that “someone” just decides which icons to include and which not, and there is something basic missing like a phone icon. 

The “core” solution should just have a very narrow set of included icons, and it should be quite easy to add custom icons. 

Even better there shouldn't be a "social icons" block anyway. It should be just be part of normal navigation behaviour.

**Comment by eric-michel** [comment]
> Even better there shouldn't be a "social icons" block anyway. It should be just be part of normal navigation behaviour.

Simply renaming the block to "Icons" and including icon options outside of just social would be welcome as well (as long as this also included the ability to easily add custom icons).

**Comment by skorasaurus** [comment]
Hi, the ability to add additional icons and services is being tracked in https://github.com/WordPress/gutenberg/pull/30749

**Comment by Mamaduka** [comment]
Closing as duplicate of #19041. Work on the features continues in #59368.
